.circleci/config.yml

```diff
@@ -54,6 +54,7 @@ jobs:
       MINIKUBE_WANTREPORTERRORPROMPT: false
       MINIKUBE_HOME: /home/circleci
       CHANGE_MINIKUBE_NONE_USER: true
+    resource_class: large
```

Comment:
Curious to know why we need a large build container (assuming it is a build container)

Reply:
This is for the integration tests. The container was increased to match the size that CircleCI has with the limits I increased minikube to. https://circleci.com/docs/2.0/configuration-reference/#resourceclass